src/components/molecules/EarthEditor/DatasetPane/DatasetModal/hooks.ts

```diff
@@ -16,7 +16,7 @@ export default (
     sheetName: string,
     schemeId: string | null,
   ) => Promise<void>,
-  onNotify?: (type: NotificationType, text: string) => void,
+  onNotify?: (type: NotificationStyleType, text: string) => void,
```

Comment:
```suggestion
  onDatasetAdd?: () => void;
```

Notification should be implemented at organisms. Molecules should not handle anything about notification.